tests/arch/arm64/arm64_high_addresses/testcase.yaml

```diff
@@ -0,0 +1,26 @@
+common:
+  filter: CONFIG_64BIT
+  arch_allow: arm64
```

Comment:
Do we need `filter: CONFIG_64BIT` if we specify `arch_allow: arm64`?

Reply:
right. This PR is so old that at that time there was no arm64 architecture